sii/models.py

```diff
@@ -23,7 +29,14 @@ class Cabecera(Schema):
 
 class PeriodoImpositivo(Schema):
     Ejercicio = fields.String(required=True)  # TODO validate Año en formato 'YYYY'
-    Periodo = fields.String(required=True)  # TODO validate Enumeration '01' para enero, '02' para febrero, ..., '0A' Periodicidad anual
+    Periodo = fields.String(required=True)
```

Comment:
You can use [OneOf](https://marshmallow.readthedocs.io/en/latest/api_reference.html#marshmallow.validate.OneOf) to do this validation as following:
```python
Periodo = fields.String(validate=validate.OneOf(PERIODO_VALUES))
```